content/docs/querying/operators.md

```diff
@@ -17,7 +23,7 @@ The following binary arithmetic operators exist in Prometheus:
 Binary arithmetic operators are defined between scalar/scalar, vector/scalar,
 and vector/vector value pairs.
 
-**Between two scalars**, the behavior is obvious: they evaluate to another
+**Between two scalars**, the behaviour is obvious: they evaluate to another
```

Comment:
So should we go with double comma in ", i.e.,"/", e.g.," then as well? Otherwise it's becoming a mixture of both.